embedded/tbtree/tbtree.go

```diff
@@ -641,7 +640,7 @@ func OpenWith(path string, nLog, hLog, cLog appendable.Appendable, opts *Options
 	}
 
 	if validatedCLogEntry == nil {
-		t.root = &leafNode{t: t, mut: true}
+		t.root = &leafNode{t: t, mut: false}
```

Comment:
why is this changed to false?

Reply:
there is just a small difference, if the node start mutated it means changes will be done in the same instance, otherwise a copy will be made. It's better to always do the changes in a copy so any rollback can be done, even when starting with a fresh btree